### DISCUSSION TITLE
onnxruntime-gpu is downgrading when updating

### DISCUSSION THREAD

**Discussion by edicam** [comfyanonymous/ComfyUI#5525] [General]
Hi,
every time I update the onnxruntime-gpu is downgraded to version 1.15, then I need to manually install an update version.
I'm not sure which is the node that is doing it, so I'm attaching the log. It happens on line 165.
[comfyui.prev.zip](https://github.com/user-attachments/files/17662575/comfyui.prev.zip)
There are several libraries that can't be downgraded. What about to include onnxruntime-gpu in this list ?

**Comment by edicam** [comment]
I'm bumping this discussion because the issue stills present.

**Comment by ltdrdata** [comment]
The inference-gpu dependency used by `yoloworld` does not support higher version packages.
(It's not even usable with Python 3.12.)
You must either uninstall yoloworld or choose the older version of onnxruntime - those are your only two options.
